keystone_json_assignment/json.py

```diff
@@ -76,56 +78,68 @@ def _get_public_id(self, user_id):
             'local_id': user_id,
             'entity_type': 'user'})
 
+    def load_user_project_map(self):
+        mtime = datetime.datetime.utcfromtimestamp(
+            os.path.getmtime('/etc/keystone/user-project-map.json'))
```

Comment:
Shouldn't we catch a potential `OSError` in case the map isn't there?

Reply:
We already weren't watching for that in the `open()` so this isn't change isn't introducing a regression, but we could add that safety check in a followup

Reply:
Cosmetic: I think the utc conversion is unnecessary. Mtime is time in seconds since 1970, aka a beautiful simple number, continuously increasing and easy to use for comparison. So we can just compare the mtime